### DISCUSSION TITLE
FaceID: new IPAdapter model for FACE

### DISCUSSION THREAD

**Discussion by kubilaykilinc** [lllyasviel/Fooocus#1579] [Ideas]
FaceID: new IPAdapter model
Best face swapping (actually best face rendering from reference image)
I hope some day this can be implemanted to fooocus.

This video for comfyui
https://www.youtube.com/watch?v=jSu_tKfg5rI

More info
https://huggingface.co/h94/IP-Adapter-FaceID

**Comment by mashb1t** [comment]
see https://github.com/lllyasviel/Fooocus/issues/1522

**Comment by jepjoo** [comment]
FaceID is SD 1.5 only, it is not compatible with Fooocus.

**Reply by IPv6** [reply]
would be cool to apply it to SD1.5 refiner during refine stage... with early switch the best of two worlds can be used, probably
